easybuild/easyconfigs/l/libevent/libevent-2.1.11.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'libevent'
+version = '2.1.11'
+
+homepage = 'https://libevent.org/'
+
+description = """
+ The libevent API provides a mechanism to execute a callback function when
+ a specific event occurs on a file descriptor or after a timeout has been
+ reached.  Furthermore, libevent also support callbacks due to signals or
+ regular timeouts.
+"""
+
+toolchain = SYSTEM
+toolchainopts = {'optarch': True}
```

@levlafayette please add 'pic' to be in line with the GCCcore based version, and remove 'optarch' because it is already True by default:
```suggestion
toolchainopts = {'pic': True}
```